projects/common/src/utilities/formatters/string/highlight.pipe.ts

```diff
@@ -5,6 +5,10 @@ import { assertUnreachable } from '../../lang/lang-utils';
 // TODO: Currently htHighlight does not escape reserved regex characters
```

Comment:
remove this TODO??

Reply:
good catch. removed it. @itssharmasandeep